injection/sharedmain/main.go

```diff
@@ -96,7 +96,13 @@ func GetLoggingConfig(ctx context.Context) (*logging.Config, error) {
 	if err := wait.PollImmediate(1*time.Second, 5*time.Second, func() (bool, error) {
 		var err error
 		loggingConfigMap, err = kubeclient.Get(ctx).CoreV1().ConfigMaps(system.Namespace()).Get(logging.ConfigMapName(), metav1.GetOptions{})
-		return err == nil || apierrors.IsNotFound(err), nil
+		if err == nil {
+			return true, nil
+		}
+		if apierrors.IsNotFound(err) || apierrors.IsTimeout(err) || apierrors.IsServerTimeout(err) {
+			return false, nil
+		}
+		return false, err
```

Comment:
So what are we trying to achieve here? Retry on NotFound/Timeout?
And you want to pass the error in the other cases?

Reply:
> So what are we trying to achieve here? Retry on NotFound/Timeout?
> And you want to pass the error in the other cases?

Yes, exactly！

Previously, we would only get the `timed out waiting for the condition` error message. Retry on NotFound. But sometimes that's not the root cause.

Refer to https://github.com/knative/operator/issues/119#issuecomment-638010432. In fact, What I encounter is forbidden Error. We lost the root cause.